errors.go

```diff
@@ -40,8 +52,20 @@ const (
 // Message returns the status code message
 func (o ErrorCode) Error() string {
 	switch o {
-	case -0x2700:
-		return fmt.Sprintf("Certificate verification failed: %d", o)
+	case -0x0001:
```

Comment:
Is it possible to refer to the consts instead of the numbers, for more verbosity?

Reply:
damn, in my defence, I did try it earlier but got errors